src/Enhavo/Bundle/DashboardBundle/Tests/Widget/Type/NumberWidgetTypeTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+
+namespace Enhavo\Bundle\DashboardBundle\Tests\Widget\Type;
+
+
+use Enhavo\Bundle\AppBundle\View\ViewData;
+use Enhavo\Bundle\DashboardBundle\Provider\Provider;
+use Enhavo\Bundle\DashboardBundle\Widget\Type\NumberWidgetType;
+use Enhavo\Component\Type\FactoryInterface;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\OptionsResolver\Exception\MissingOptionsException;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class NumberWidgetTypeTest extends TestCase
+{
+    public function testCreateViewData()
+    {
+        $dependencies = $this->createDependencies();
+        $dependencies->factory->method('create')->willReturnCallback(
+            function () {
+                $providerMock = $this->createProviderMock();
+                $providerMock->method('getData')->willReturn(543);
+
+                return $providerMock;
+            }
+        );
+        $instance = $this->createInstance($dependencies);
+
+        $viewData = new ViewData();
+        $options = [
+            'provider' => [
+                'type' => 'type'
+            ]
+        ];
+        $instance->createViewData($options, $viewData);
+        $this->assertEquals(543, $viewData->get('value'));
+    }
+
+    public function testGetName()
+    {
+        $dependencies = $this->createDependencies();
+        $instance = $this->createInstance($dependencies);
+
+        $this->assertEquals('number', $instance->getName());
```

Comment:
`getName` is a static function. You can reduce your code to one line

`$this->assertEquals('number', NumberWidgetTypeTest::getName())`